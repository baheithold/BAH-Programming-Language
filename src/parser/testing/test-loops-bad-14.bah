#This file tests an invalid loop


while (i < 10)

}

