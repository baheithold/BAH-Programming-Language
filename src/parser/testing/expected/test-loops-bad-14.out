ERROR, line 6: expected CBRACE, but encountered END_OF_INPUT
